bot/discordbot.py

```diff
@@ -319,6 +319,21 @@ async def on_message(message):
 				.replace('l', 'w')\
 				.replace('R', 'W')\
 				.replace('L', 'W')\
+				.replace('na', 'nya')\
+				.replace('ne', 'nye')\
+				.replace('ni', 'nyi')\
+				.replace('no', 'nyo')\
+				.replace('nu', 'nyu')\
+				.replace('Na', 'Nya')\
+				.replace('Ne', 'Nye')\
+				.replace('Ni', 'Nyi')\
+				.replace('No', 'Nyo')\
+				.replace('Nu', 'Nyu')\
+				.replace('NA', 'NYA')\
+				.replace('NE', 'NYE')\
+				.replace('NI', 'NYI')\
+				.replace('NO', 'NYO')\
+				.replace('NU', 'NYU')\
```

Comment:
please use regex and also check the length of the string before and after replacing that so it doesnt go above 2000 length